### PR TITLE
Refactor organisation-related endpoints

### PR DESCRIPTION
**Pull request by boffbowsh**
Move the Organisation index into a separate controller as it’s
unrelated to anonymous feedback
Add an Organisation show action to retrieve the title etc for a single
organisation
Add a spec for the org summary action

Backwards-compatible for the support app as the `/anonymous-feedback/organisations` endpoint still works, but should be removed.

### CODE REVIEW

File: app/controllers/anonymous_feedback/organisations_controller.rb [hunk 1]
```diff
@@ -12,6 +7,12 @@ def show
         ordering = "last_7_days"
       end
       organisation = Organisation.find_by(slug: params[:slug])
+
+      if organisation.nil?
+        render nothing: true, status: :not_found
+        return
+      end
```
Comment (mikejustdoit) [inline app/controllers/anonymous_feedback/organisations_controller.rb]: This (the part of the spec that covers it) could go in a separate commit if they're as unrelated to the other changes as it seems

Comment (boffbowsh) [inline app/controllers/anonymous_feedback/organisations_controller.rb]: I did it all as one as I wrote the spec from scratch - it didn't exist before.

**Comment by mikejustdoit** [comment]
Looks good
